src/Compilers/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -211,6 +297,9 @@ static Microsoft.CodeAnalysis.CSharp.SyntaxFactory.WithExpression(Microsoft.Code
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.ArgumentList.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentListSyntax
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.Initializer.get -> Microsoft.CodeAnalysis.CSharp.Syntax.InitializerExpressionSyntax
 *REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ObjectCreationExpressionSyntax.NewKeyword.get -> Microsoft.CodeAnalysis.SyntaxToken
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ParameterSyntax.AttributeLists.get -> Microsoft.CodeAnalysis.SyntaxList<Microsoft.CodeAnalysis.CSharp.Syntax.AttributeListSyntax>
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ParameterSyntax.Modifiers.get -> Microsoft.CodeAnalysis.SyntaxTokenList
+*REMOVED*Microsoft.CodeAnalysis.CSharp.Syntax.ParameterSyntax.Type.get -> Microsoft.CodeAnalysis.CSharp.Syntax.TypeSyntax
```

Comment:
The removal looks suspicious. Could this be a source of a binary break?

Reply:
We add an override instead (lines 128-130). Since we always prefer to emit `callvirt` instructions, it should be fine for most use cases. We did the same change when we introduced BaseObjectCreationExpressionSyntax as a new parent node.

---
In reply to: [451226984](https://github.com/dotnet/roslyn/pull/45684#discussion_r451226984) [](ancestors = 451226984)